### ISSUE TITLE
Log does not work after updating to Serenity 4

### ISSUE THREAD

**Issue by Iandei** [serenity-bdd/serenity-core#3269]
### What happened?

I work with serenity and gradle and after updating serenity to version 4.0.12, the tests are working but the log is not writting the steps

### What did you expect to happen?

The steps appear in console log

### Serenity BDD version

4.0.12

### JDK version

18

### Execution environment

Im using Chrome

### How to reproduce the bug.

Update to last version of serenity and run any test

### How can we make it happen?

Hire the Serenity BDD team

**Comment by wakaleo** [comment]
Could you give an example of the logs you saw previously, and what you see now?

**Comment by Iandei** [comment]
I have taken the test to the same point and this is the difference. I leave 2 photos so you can see the difference:

Before: 
![image](https://github.com/serenity-bdd/serenity-core/assets/146440615/4c492e70-86c2-4fe8-98ac-df9cf6c4771a)

After:
![image](https://github.com/serenity-bdd/serenity-core/assets/146440615/44f6ded6-a7bb-45be-8989-d45cac442017)

**Comment by wakaleo** [comment]
Try setting `serenity.logging` to VERBOSE

**Comment by Iandei** [comment]
I have it, this is part of my serenity.conf:
serenity{
  take.screenshots=FOR_FAILURES
  console{
    colors=true
    timestamp=true
  }
  current.target.version = 1.0.4
  logging=VERBOSE
}

**Comment by AndresCoGGi** [comment]
Can you find anything solutions ?

**Comment by AndresCoGGi** [comment]
I have the same problem

**Comment by Iandei** [comment]
No, Im still waiting for a solution...Im still using serenity 3.9.8

**Comment by wakaleo** [comment]
I had . Remember this is an open source projects, so waiting for a solution is not the fastest way to get a solution - the fastest way is to look in the code and understand what is not working. The class that does this is ConsoleLogListener. It appears to be getting called correctly. The logging is done via slf4j, so it might be an slf4j config issue. I won't have time to look further very soon.

**Comment by Iandei** [comment]
ok, thanks wakaleo! I will investigate on my own and let all the community know if I get something special. If you get the solution let me know

**Comment by SrivaniChennupati** [comment]
@landei : did u find solution for this as I am facing similar issue after upgrading to serenity 4

**Comment by SrivaniChennupati** [comment]
@wakaleo : Could you able to give your insights on this please , facing similar issue

**Comment by Iandei** [comment]
We continue without native logs, but I found a temporary solution using logback. I added this dependencies on my build.gradle:

`implementation 'ch.qos.logback:logback-core:1.5.15'
 testImplementation 'ch.qos.logback:logback-classic:1.5.15'`

and I have created a file called "logback.xml" as this:

`<configuration>
    <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
        <encoder>
            <pattern>%d{HH:mm:ss.SSS} %msg%n</pattern>
        </encoder>
    </appender>

    <root level="info">
        <appender-ref ref="CONSOLE" />
    </root>
</configuration>`

**Comment by dmiales** [comment]
use testImplementation 'org.slf4j:slf4j-simple:2.0.16'
